e2e/perf/get-pr-info.py

```
@@ -0,0 +1,189 @@
+    if workflow_run.get('event') != 'pull_request':
+        return []
+
+    head_branch = workflow_run.get('head_branch')
+    if not head_branch:
+        return []
+
+    print(f"Searching for PRs by head_branch: {head_branch}", file=sys.stderr)
+
+    # Use gh CLI to search for PRs by head branch
+    try:
+        result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)